cmd/convert.go

```diff
@@ -131,6 +134,15 @@ func objectToTemplate(objects *[]runtime.RawExtension, templateLabels *map[strin
 			return fmt.Errorf(fmt.Sprintf("Failed to marshal Raw resource back to YAML\n%v\n", updatedJSON) + err.Error())
 		}
 
+		if m[k8sR.GetKind()] == nil  {
+			m[k8sR.GetKind()] = data
+
+		} else {
+			newdata:=append(m[k8sR.GetKind()],seperator...)
+			data=append(newdata,data...)
+			m[k8sR.GetKind()] = data
+		}
+
 		tf := chart.File{
```

Comment:
We should move the File creation logic into a second loop. With a single loop, we create extra File objects that I assume get overwritten, but seems like it could cause issues later.

IE, the first loop over the template objects will aggregate per kind, then a second loop over map `m` that writes our one file per type.